tests/test_scalar_multiplication.py

```diff
@@ -50,3 +51,23 @@ def test_scalar_inverse(x: Vector, scalar: float):
 @given(x=vectors(), scalar=floats())
 def test_scalar_rmul(x: Vector, scalar: float):
     assert scalar * x == x.scale_by(scalar)
+
+
+@given(x=vectors(), scalar=st.integers())
+def test_integer_multiplication(x: Vector, scalar: int):
+    assert scalar * x == float(scalar) * x
+
+
+@given(x=vectors(), scalar=st.integers())
+def test_integer_division(x: Vector, scalar: int):
+    assume(scalar != 0)
+    assert x / scalar == x / float(scalar)
+
+
+@given(x=vectors())
+def test_division_by_zero(x: Vector):
+    with raises(ZeroDivisionError):
+        x / 0
+
+    with raises(ZeroDivisionError):
```

Comment:
Why twice?

Reply:
With integer `0` and floating-point `0.`:
```
>>> type(0), type(0.)
(int, float)
```

Reply:
Oh! Missed the dot.